sections/image-banner.liquid

```diff
@@ -45,7 +45,7 @@
           assign sizes = "100vw"
         endif
       -%}
-      {{ section.settings.image | image_url: width: 1500 | image_tag:
+      {{ section.settings.image | image_url: width: 3840 | image_tag:
```

Comment:
Here I believe we can keep 3840 as a biggest value among all available widths (line 54). Or we could replace it with `section.settings.image.width ` which gives us the same result. I agree that we don't want to have a limit of 1500 since most uploaded images have bigger resolution which caused a blurry effect.

Reply:
The reasoning here is that when using this way of outputting images we tend to mention sizes in two different spots. 

- Once in the `image_url: width: XXX`, which is a fallback in case the browser doesn't support the `srcset` attribute. And it also serves as the image we will be querying/resizing in `srcset`.
- Once in the `srcset` attribute. This will use the image mentioned in the point above and its size to output the different sizes. So that's why we always need to make sure that the size is as big as our biggest output value in the `srcset`.